lib/miyuki/version.rb

```diff
@@ -13,5 +13,5 @@
 ##
 
 module Miyuki
-  VERSION = '0.5.8.1'
+  VERSION = '0.5.8.2'
```

Comment:
This is ok, but usually is good practice leaving to the manteiner the bump of the version :dancer: